internal/pkg/otel/README.md

```diff
@@ -35,32 +35,50 @@ This section provides a summary of components included in the Elastic Distributi
 
 | Component | Version |
 |---|---|
+| [filelogreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/filelogreceiver/=&gt; github.com/elastic/opentelemetry-collector-contrib/receiver/filelogreceiver v0.0.0-20250317163643-19cd4e80024f/receiver/filelogreceiver/README.md) | =&gt; github.com/elastic/opentelemetry-collector-contrib/receiver/filelogreceiver v0.0.0-20250317163643-19cd4e80024f |
```

Comment:
Having this twice seems like an artifact of the replace directive, it's not entirely wrong but it is unnecessary duplication.